### PR TITLE
[ID] Remove minikube instalasi guide and add index untuk tools to redirect to original minikube install

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR address one issue as mentioned here #26955

/cc @kubernetes/sig-docs-id-reviews

### CODE REVIEW

File: content/id/docs/tasks/tools/_index.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+Setelah kamu memiliki `minikube` yang bekerja, kamu bisa menggunakannya
+untuk  [menjalankan aplikasi contoh](/id/docs/tutorials/hello-minikube/).
+
+## kubeadm
+
+Kamu dapat menggunakan {{< glossary_tooltip term_id="kubeadm" text="kubeadm" >}} 
+untuk membuat dan mengatur klaster Kubernetes.
+`kubeadm` menjalankan tugas-tugas yang diperlukan untuk mendapatkan minimum
+keaktifan, keamanan dari klaster yang hidup dan berjalan secara yang mudah
+bagi pengguna.
```
Comment (habibrosyad) [inline content/id/docs/tasks/tools/_index.md:62]:
```suggestion
`kubeadm` menjalankan langkah-langkah yang diperlukan untuk mendapatkan klaster
dengan kelaikan dan keamanan minimum, aktif dan berjalan dengan cara yang mudah
bagi pengguna.
```

File: content/id/docs/tasks/tools/_index.md [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+
+[Instalasi kubeadm](/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) memperlihatkan tentang bagaimana melakukan instalasi kubeadm.
+Setelah terinstall, kamu dapat menggunakannya untuk [mebuat klaster](/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
```
Comment (habibrosyad) [inline content/id/docs/tasks/tools/_index.md:65]:
```suggestion
Setelah terinstal, kamu dapat menggunakannya untuk [membuat klaster](/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
```

**Review by habibrosyad — COMMENTED** [review]
Thanks for the PR @ariscahyadi. I have some suggestions below regarding the updated content.

Also, in addition to the update in this PR you also need to:
1. Remove https://github.com/kubernetes/website/blob/master/content/id/docs/tasks/tools/install-minikube.md (as this is removed in the upstream and should also be the goal of this PR)
2. Add redirection as what EN and JA team do with the page (xref [sample redirection](https://github.com/nate-double-u/kubernetes-website/blob/3ae3c7c7e7c1b6740869cc11702fff66919d6302/static/_redirects#L288),  [sample PR](https://github.com/kubernetes/website/pull/24416/files))

I'm expecting there should be 3 files being changed here (xref https://github.com/kubernetes/website/pull/24416/files).

**Comment by ariscahyadi** [comment]
Thanks @habibrosyad for your comments. I have already addressed all of them.

**Comment by habibrosyad** [comment]
/assign @reylejano 
for the final approval since there is an update in the redirects

**Comment by reylejano** [comment]
@habibrosyad Since this is an ID localization PR, I'll assign @irvifa 

/assign  @irvifa

**Comment by habibrosyad** [comment]
/assign @irvifa 

It seems the previous command does nothing ¯\_(ツ)_/¯

**Review by irvifa — COMMENTED** [review]
/approve

**Pull request merged**